core/src/test/java/com/datastax/oss/driver/internal/core/metadata/MetadataManagerTest.java

```diff
@@ -308,7 +308,7 @@ private void waitForPendingAdminTasks() {
     // This works because the event loop group is single-threaded
     Future<?> f = adminEventLoopGroup.schedule(() -> null, 5, TimeUnit.NANOSECONDS);
     try {
-      Uninterruptibles.getUninterruptibly(f, 100, TimeUnit.MILLISECONDS);
+      Uninterruptibles.getUninterruptibly(f, 500, TimeUnit.MILLISECONDS);
```

Comment:
Are we waiting for some specific event to occur? If so, would it be possible to use Awaitility to wait on that condition?

Reply:
I'm not sure,but I'll investigate further. With the timeout set to 100, I couldn't reproduce the failure locally, and it failed sporadically on Jenkins/CloudBees.
I can lower the value and reproduce it and figure out what I need to be waiting on.